presences/src/main/resources/sql/044-MA-967-count-alert-by-line.sql

```diff
@@ -0,0 +1,19 @@
+-- Delete old data due to the modification of the table
+DELETE FROM presences.alerts WHERE 1 = 1;
```

Comment:
tu peux juste utiliser truncate table no?

Reply:
Yes
Et si tu veux juste delete, pas besoin de la conditionner a ma connaissance ?

Reply:
yes, mais tu te prend un warning en mode "attention y'a pas de condition donc tu vas tout supprimer"

Reply:
The WHERE clause specifies which record(s) should be deleted. If you omit the WHERE clause, all records in the table will be deleted!

Donc non t'as raison Jin !

Reply:
ouai cette phrase là

Reply:
DELETE FROM presences.alerts = Unsafe query: 'Delete' statement without 'where' clears all data in the table
TRUNCATE TABLE presences.alerts ☑

Reply:
> yes, mais tu te prend un warning en mode "attention y'a pas de condition donc tu vas tout supprimer"

Mais du coup c'est ce qu'on veut, non ?